R/MsBackendDataFrame.R

```diff
@@ -95,80 +96,79 @@ setReplaceMethod("collisionEnergy", "MsBackendMemory", function(object, value) {
 })
 
 #' @rdname hidden_aliases
-setMethod("fromFile", "MsBackendMemory", function(object) {
+setMethod("fromFile", "MsBackendDataFrame", function(object) {
     if (any(colnames(object@spectraData) == "fromFile"))
         object@spectraData$fromFile
     else rep(NA_integer_, times = length(object))
 })
 
 #' @rdname hidden_aliases
-setMethod("intensity", "MsBackendMemory", function(object) {
+setMethod("intensity", "MsBackendDataFrame", function(object) {
     if (any(colnames(object@spectraData) == "intensity"))
         object@spectraData$intensity
     else {
-        lst <- list(numeric)
+        lst <- list(numeric())
```

Comment:
Why not 
```r
rep(list(numeric()), times = length(object))
```

Reply:
Good point. Thanks. Thought that one would be slower because it creates a new object each time - but was wrong. I'll change.